.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add `.classpath` to `.gitignore`. `.gitignore` is a file located in the root directory of the project. If eclipse doesnt show it just use gitbash and use vim from there to edit it.